### PR TITLE
Set default PDA uplink and music visibility to false

### LINKED ISSUE #27376: When PDA fails to load properly, you can see uplink options + play music
## Description

None of the buttons actually work and nothing loads, but it's quite funny to have the options visible even if your PDA is not intended to do either of those things

**Reproduction**

Haven't found any reproduction steps, but other people have reported the same issue

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by ShadowCommander**
## About the PR

Fixes #27376 

Sometimes the PDA shows uplink and music when opened. I've added logging to check if it's state arriving before the PDA menu is opened.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- fix: Fixed PDA sometimes showing uplink and music buttons when the PDA was not able to use them.

### CODE REVIEW

File: Content.Client/PDA/PdaBoundUserInterface.cs [hunk 1]
```diff
@@ -4,18 +4,20 @@
...
+
         [ViewVariables]
         private PdaMenu? _menu;
 
         public PdaBoundUserInterface(EntityUid owner, Enum uiKey) : base(owner, uiKey)
         {
+            _pdaSystem = EntMan.System<PdaSystem>();
         }
 
         protected override void Open()
```
Comment (deltanedas) [inline Content.Client/PDA/PdaBoundUserInterface.cs:23]: i think this should be if (_menu == null) { create it } _menu.OpenCenteredLeft();

it fixed similar issues in other things so might fix this as well

**Comment by metalgearsloth** [comment]
Doesn't setting it to false just bandaid an underlying issue? The state should already be received before they even touch the BUI.

**Comment by ShadowCommander** [comment]
This is a bandaid yes. I wasn't sure on how the other UIs handled this, so putting a bandaid and logging an error when it happened seemed the fastest way to check if this is the actual problem or if it's something else, because we can't reproduce it on local.

File: Content.Client/PDA/PdaBoundUserInterface.cs (hunk 1 above)
Comment (ShadowCommander) [reply Content.Client/PDA/PdaBoundUserInterface.cs:23]: Do you have any examples?

Comment (deltanedas) [reply Content.Client/PDA/PdaBoundUserInterface.cs:23]: salv magnet

Comment (AJCM-git) [reply Content.Client/PDA/PdaBoundUserInterface.cs:23]: just use the ??= operator

**Pull request merged**